src/lucky/memoizable.cr

```diff
@@ -22,7 +22,7 @@ module Lucky::Memoizable
   macro memoize(method_def)
     {% raise "You must define a return type for memoized methods" if method_def.return_type.is_a?(Nop) %}
     {%
-      raise "All arguments must have an explicit type for memoized methods" if method_def.args.any? &.is_a?(Nop) # ameba:disable Style/IsAFilter
+      raise "All arguments must have an explicit type for memoized methods" if method_def.args.any? &.is_a?(Nop)
```

Comment:
Had to add this to get the specs green. Apparently Ameba updated since our last PR was merged.

Reply:
Sweet! Yeah, I had filed a bug. Looks like it was updated 👍